org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  curBucket:
+    Type: String
+Resources:
+  MonthlyCostReport:
+    Type: "AWS::CUR::ReportDefinition"
+    Properties:
+      ReportName: !Sub ""
+      ReportVersioning: False
+      TimeUnit: "hourly"
```

Comment:
I think TimeUnit should be MONTHLY, not HOURLY.  I recall that I first made an HOURLY report.  At the end of the month the report was massive, spanning multiple files.  Changing to MONTHLY provided all the needed information in one file that was just a few MB.